Tools/gulp/tasks/compileExtensions.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs-extra');
+const utils = require('../utils');
+
+
+
+async function compileSpellfix(dest) {
+	try {
+		await utils.execCommand(`wget -nc -q -O ${dest}/sqlite.tar.gz "https://www.sqlite.org/src/tarball/sqlite.tar.gz?r=release"`);
+	} catch (e) {
+		// console.info(e);
+	}
+
+	try {
+		await utils.execCommand(`wget -nc -O ${dest}/amalgamation.tar.gz "https://www.sqlite.org/2020/sqlite-autoconf-3330000.tar.gz"`);
+	} catch (e) {
+		// console.info(e);
```

Comment:
Any reason why the exceptions are silenced? If it fails to download, the script should end there since it won't be able to compile. So in fact I think you don't need a try/catch block here.